test/utility-test.js

```diff
@@ -6,7 +6,7 @@ var rc_util = require('../src/lib/utility.js');
 var invalid_crawl = require('./data/invalid_crawl.json');
 var valid_crawl = require('./data/valid_crawl.json');
 
-var db_url = 'postgres://postgres:postgres@127.0.0.1:5432/circle_test';
+var db_url = '52.88.154.28:9090'; // testing engine might not have access to this url.
```

Comment:
but we are using it in .skipped functions. should I remove skipped functions as well ?